ecoscope/io/earthranger.py

```diff
@@ -683,6 +714,10 @@ def get_patrol_events(self, since=None, until=None, patrol_type=None, status=Non
         events_df = clean_time_cols(events_df)
 
         events_df["geometry"] = events_df["geojson"].apply(lambda x: shape(x.get("geometry")))
+        events_df["time"] = events_df["geojson"].apply(
+            lambda x: datetime.datetime.strptime(x.get("properties").get("datetime"), "%Y-%m-%dT%H:%M:%S%z")
+        )
+
```

Comment:
I feel like we eventually (not necessarily with this PR) want this logic to be a utility function like `clean_time_cols` but for parsing the geojson we get from EarthRanger. 

I'm not sure if it's something that _can_ happen within EarthRanger, but I think it's worth surfacing an error message that indicates bad/missing data if these lambdas were to fail. Rather than the `AttributeError` or `TypeError` that would materialism currently. 

I don't think this needs to be addressed with this PR, just writing this here for posterity.

Reply:
Yea, I agree. Thanks Alex!